Tests/KituraNetTests/RegressionTests.swift

```diff
@@ -166,28 +166,26 @@ class RegressionTests: KituraNetTest {
     /// response with `Connection: Close` header.
     /// This is to verify the fix introduced in Kitura-net PR #229, where a malformed
     /// request sent during a Keep-Alive session could cause the server to crash.
-    func testBadRequestFollowingGoodRequest() {
-       do {
-            let server: HTTPServer = try startServer(nil, port: 0, useSSL: false, allowPortReuse: false)
-            defer {
-                server.stop()
-            }
-
-            guard let serverPort = server.port else {
-                XCTFail("Server port was not initialized")
-                return
-            }
-            XCTAssertTrue(serverPort != 0, "Ephemeral server port not set")
-
-            var goodClient = GoodClient(with: HTTPClient(with: self.expectation(description: "Bad request error")))
-            try! goodClient.makeGoodRequestFollowedByBadRequest(serverPort)
-            waitForExpectations(timeout: 10)
-
-        } catch {
-            XCTFail("Couldn't start server")
-        }
-
-    }
+    //func testBadRequestFollowingGoodRequest() {
```

Comment:
We don't need to comment out the code of the test, just the entry in the test array.

Reply:
I think that doesn't work on macOS. I think Xcode doesn't pick the tests from the array, it runs all tests found in a file. I may be wrong - just trying a travis CI run.

Reply:
I'm sorry, you were right!